### DISCUSSION TITLE
DataSheet styles broken in a flexible container

### DISCUSSION THREAD

**Discussion by Quienx** [nick-keller/react-datasheet-grid#59] [Q&A] [answered]
i have this error when a try to use the DataSheet in a grid or a flex container, but the error does not occur in a normal HTML  data flow, sorry for my english

visual example:
https://www.loom.com/share/ff3f7af716a64feab36bddff059456ac

**Comment by nick-keller** [comment, marked as answer]
Hello @YordinsonPolar,
Can you try to wrap your `<DataSheetGrid/>` in a div? You can then add CSS to this div like a width, max width, flex... 
The CSS would depend on the situation, but the idea is to make the width of the container not depend on its content.

**Comment by Quienx** [comment]
ok, I wrap the  <DataSheetGrid/> in a div and use some styles like width:100% and overflow:hidden and the problem solve, thank you.
